app/src/main/java/by/evgen/an11pazniakhomework14/view/MainActivity.kt

```diff
@@ -0,0 +1,286 @@
+//    private fun printAboutCocktail(number: Int) {
+//        drinkTextView = findViewById<TextView>(R.id.textViewDrinkResult)
+//        val textToView: String
+//        when (val spirit: Spirits = OrderList.listOfDrinks[number]) {
+//            is WhiskeyCola -> {
+//                spirit.price = whiskeyColaPrice
+//                textToView =
+//                    spirit.getInfo() + "\n" + spirit.drinkWithCoworkers() + "\n" + spirit.drinkWithFriends() + "\n" + spirit.drinkAlone() + "\n" + "The price ${spirit.name} with taxes: " + spirit.price
+//                drinkTextView.text = textToView
+//            }
+//            is Jorsh -> {
+//                spirit.price = jorshPrice
+//                textToView =
+//                    spirit.getInfo() + "\n" + spirit.drinkToFlyAway() + "\n" + spirit.drinkLikeACure() + "\n" + spirit.drinkAlone() + "\n" + "The price ${spirit.name} with taxes: " + spirit.price
+//                drinkTextView.text = textToView
+//            }
+//            is WhiteRussian -> {
+//                spirit.price = whiteRussianPrice
+//                textToView =
+//                    spirit.getInfo() + "\n" + spirit.drinkToFeelAwesome() + "\n" + spirit.drinkLikeACure() + "\n" + spirit.drinkAlone() + "\n" + "The price ${spirit.name} with taxes: " + spirit.price
+//                drinkTextView.text = textToView
+//            }
+//            is Whiskey -> {
+//                spirit.price = whiskeyPrice
+//                textToView =
+//                    spirit.getInfo() + "\n" + spirit.drinkWithFriends() + "\n" + spirit.drinkAlone() + "\n" + "The price ${spirit.name} with taxes: " + spirit.price
+//                drinkTextView.text = textToView
+//            }
+//            is Vodka -> {
+//                spirit.price = vodkaPrice
+//                textToView =
+//                    spirit.getInfo() + "\n" + spirit.drinkLikeACure() + "\n" + spirit.drinkAlone() + "\n" + "The price ${spirit.name} with taxes: " + spirit.price
+//                drinkTextView.text = textToView
+//            }
+//        }
+//    }
+
+//    private fun delete() {
+//        val removeButton: Button = findViewById(R.id.buttonDelete)
+//        removeButton.setOnClickListener {
+//            drinkTextView = findViewById(R.id.textViewDrinkResult)
+//            if ((OrderList.listOfDrinks.size > 0) && (drinkTextView.text.isNotEmpty())) {
+//                OrderList.listOfDrinks.removeAt(iter)
+//            }
+//            drinkTextView.text = ""
+//        }
+//    }
+
+    private fun showCatalog() {
+        catalogTextView = findViewById(R.id.textViewCatalogContent)
+        var catalogText = ""
+        for (spirit in catalogClass.spirits) {
+            catalogText = catalogText.plus(getInfoAboutDrink(spirit))
+        }
+        catalogTextView.text = catalogText
+    }
+
+    private fun getInfoAboutDrink(spiritArg: Spirits): String {
+        var result = ""
+        when (val spirit: Spirits = spiritArg) {
```

Comment:
result = when (val spirit: Spirits = spiritArg) {

Reply:
Simplified by little other way. Thank you.